src/Locator/ReceiverLocator.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace KaroIO\MessengerMonitorBundle\Locator;
+
+use KaroIO\MessengerMonitorBundle\Exception\ReceiverDoesNotExistException;
+use Symfony\Component\Messenger\Transport\Receiver\ReceiverInterface;
+use Symfony\Contracts\Service\ServiceProviderInterface;
+
+/**
+ * @internal
+ */
+class ReceiverLocator
+{
+    private $receiverLocator;
+    private $receiverNames;
+
+    public function __construct(ServiceProviderInterface $receiverLocator, $receiverNames = [])
```

Comment:
Does `$receiverNames` need to be optional? And should it have an `array` type hint?